gcr/ansible-vault.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Let's call this `ensure-ansible-vault-installed.sh`, similar to `ensure-gcloud-installed.sh`

Reply:
done!